src/nv_ingest/util/pdf/pdfium.py

```diff
@@ -188,3 +198,219 @@ def pdfium_pages_to_numpy(
         images.append(img_arr)
 
     return images, padding_offsets
+
+
+def convert_pdfium_position(pos, page_width, page_height):
+    """
+    Convert a PDFium bounding box (which typically has an origin at the bottom-left)
+    to a more standard bounding-box format with y=0 at the top.
+    """
```

Comment:
I'm not sure this method is generally correct, as the PDF spec allows for a pretty arbitrary coordinate system.
Indeed, the origin typically is the bottom left, but IIRC the coordinate system can be laid out between any opposite corners in principle. Also, the coordinate system does not necessarily have to start at 0, it could be any arbitrary value.

Related: https://github.com/pypdfium2-team/pypdfium2/discussions/284